### DISCUSSION TITLE
BSEED TS0601 cover switch not working --> manufacturer name '_TZE204_bdblidq3' is NOT supported

### DISCUSSION THREAD

**Discussion by Piscator74** [Koenkk/zigbee2mqtt#22718] [Devices]
About this device: https://www.zigbee2mqtt.io/devices/BSEED_TS0601_cover.html
I have a couple of these devices working, but one of the devices in my latest order gave the error "manufacturer name '_TZE204_bdblidq3' is NOT supported".

The fix is relatively simple, I added a line to **bseed.js** with the new manufacturer name.
```
        fingerprint: [
		    { modelID: 'TS0601', manufacturerName: '_TZE200_yenbr4om' },
                    { modelID: 'TS0601', manufacturerName: '_TZE204_bdblidq3' },
		],
```

However, I do not know how to make a pull request for that. In repo "https://github.com/Koenkk/zigbee-herdsman-converters", I can only find a **bseed.ts** Am I supposed to update that?

**Comment by Koenkk** [comment]
Added, thanks!
